working/templates/srmfunction/SrmFunction.sln

```diff
@@ -34,6 +34,27 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Actions", "Actions", "{8A00
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "$SCRIPTNAME$_1", "$SCRIPTNAME$_1\$SCRIPTNAME$_1.csproj", "{C99E3ABD-DE32-45FA-9FB7-8C9B659E11C1}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Functions", "Functions", "{0A65DF85-1067-44AA-B14B-451BEF792C9D}"
```

Comment:
GUID 0A65DF85-1067-44AA-B14B-451BEF792C9D should be added to the guids list in the template.json file